### DISCUSSION TITLE
History in revanced YouTube

### DISCUSSION THREAD

**Discussion by THORNISH5811** [ReVanced/revanced-discussions#656] [Question] [answered]
In revanced YouTube my history is not showing I have to go in the history section to see it

**Comment by oSumAtrIX** [comment]
Please explain your issue more detailed.

**Reply by demonic-avenger** [reply]
Alright I'm definitely having an issue, and it's app related.
You see, I'm using the magisk module version of revanced.
And when I go the history section, my watch history isn't showing up.
The ones from the PC shows up just fine, but ones from the app are just not there.
It's like app's video history doesn't get saved.
And my history is enabled, I checked.

**Reply by oSumAtrIX** [reply]
Check unpatched app

**Comment by ghost** [comment, marked as answer]
Yep, explain your issue, we can't know how to help without explaining.

You can try: See if vanced microG is running on background, if history is disabled on settings, and if you're using a adblocker/private dns whitelist `s.youtube.com`.
